stages/org.osbuild.coreos.squashfs.meta.json

```
@@ -0,0 +1,28 @@
+{
+  "summary": "CoreOS squashfs needs`.",
+  "description": [
+    "This stage implements necessary CoreOS adjustments,",
+    "including creating links and modifying the read-only flag",
```

maybe a little more context

```suggestion
    "including creating hardlinks in the /boot/ filesystem and modifying the read-only flag",
```